internals/overlord/checkstate/checkers.go

```diff
@@ -202,3 +204,7 @@ type detailsError struct {
 func (e *detailsError) Details() string {
 	return e.details
 }
+
+func (e *detailsError) Unwrap() error {
```

Comment:
This isn't actually needed, but it was useful during debugging and I realised this error-wrapper type should probably have an Unwrap method, so I kept it.